_includes/sections/live-operating-systems.html

```diff
@@ -4,13 +4,15 @@ <h1 id="live_os" class="anchor"><a href="#live_os"><i class="fas fa-link anchor-
 
   {% include card.html color="success"
   title="Tails"
+  labels="warning:contrib:This software may depend on or recommend non-free software."
   image="/assets/img/tools/Tails.png"
   url="https://tails.boum.org/"
-  description="Tails is a live operating system, that starts on almost any computer from a DVD, USB stick, or SD card. It aims at preserving privacy and anonymity, and helps to: Use the Internet anonymously and circumvent censorship; Internet connections go through the Tor network; leave no trace on the computer; use state-of-the-art cryptographic tools to encrypt files, emails and instant messaging."
+  description='Tails is a live operating system, that starts on almost any computer from a DVD, USB stick, or SD card. It aims at preserving privacy and anonymity, and helps to: Use the Internet anonymously and circumvent censorship; Internet connections go through the Tor network; leave no trace on the computer; use state-of-the-art cryptographic tools to encrypt files, emails and instant messaging.'
```

Comment:
Why the change of " to ' ?

Reply:
I'd also like to know this? Seems inconsistent.

```suggestion
  description="Tails is a live operating system, that starts on almost any computer from a DVD, USB stick, or SD card. It aims at preserving privacy and anonymity, and helps to: Use the Internet anonymously and circumvent censorship; Internet connections go through the Tor network; leave no trace on the computer; use state-of-the-art cryptographic tools to encrypt files, emails and instant messaging."
```

Reply:
Should be fixed in later commit. :)

Thanks!